src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Referral.kt

```diff
@@ -71,7 +72,7 @@ class Referral(
 
   var relevantSentenceId: Long? = null,
 
-  @OneToOne(mappedBy = "referral", optional = true) @Fetch(JOIN) var actionPlan: ActionPlan? = null,
+  @OneToMany(fetch = FetchType.LAZY) @JoinColumn(name = "referral_id") var actionPlans: MutableList<ActionPlan>? = null,
```

Comment:
we have a `idx_action_plan_referral_id` "must be unique" constraint on the DB -- can we remove that too, here?

Reply:
ah missed this - thanks